### ISSUE TITLE
Set first visible time in timepicker.

### ISSUE THREAD

**Issue by SemBakkum** [mengxiong10/vue2-datepicker#176]
First let me thank you for the plugin!

There is one feature that would be nice to have and I can't find it in the docs.

Now I have the current timeOptions: 

```javascript
timeOptions: {
        start: '00:00',
        step: '00:05',
        end: '23:55'
      },
```

Opening the timepicker the first time you'll see is 00:00. Is there an option to let the first visible time to be 07:00 for example?

**Comment by mxmvshnvsk** [comment]
Maybe set your initial value like 'new Date().setHours(7, 0, 0)' and bind to v-model of datepicker component?

**Comment by SemBakkum** [comment]
Hii mxmvshnvsk, already tried that one before coming here ;) It does (pre) select a time for you, but when you open the timepicker the first time you'll see is 00:00. When you scroll down you'll see that 07:00 is selected, but i want the user to see 07:00 as the first time.

How it is now when you open the timepicker: 
<img width="231" alt="screen shot 2018-10-05 at 08 35 01" src="https://user-images.githubusercontent.com/9844595/46519794-f31a8680-c879-11e8-877e-acabca9a51eb.png">

How I would like it to be when you open it:
<img width="231" alt="screen shot 2018-10-05 at 08 35 13" src="https://user-images.githubusercontent.com/9844595/46519824-0cbbce00-c87a-11e8-9ab9-25f4716bb89b.png">

But everything before 07:00 should still be available.

Any help on this would be really appreciated.

**Comment by mengxiong10** [comment]
The selected should be in view. which component version and brower you used.
Now I am on vacation, and I'll add an event to do something when the panel changes in two days.

**Comment by SemBakkum** [comment]
Hii @mengxiong10, I'm using component version ^2.4.1 and it is in chrome v69.

Enjoy your vacation!

**Comment by mxmvshnvsk** [comment]
@SemBakkum, I think, that I have the same issue.

**Comment by mengxiong10** [comment]
The selected item should scroll into view. Try upgrading to the lastest version.

**Comment by mengxiong10** [comment]
When the type is `time`, it has a bug.
I fixed it in v2.5.0.

**Comment by SemBakkum** [comment]
@mengxiong10 it works! Thanks for the fix, going to close this issue now.
